### PR TITLE
Add documentation to close and logout methods

### PR DESCRIPTION
**Pull request by oed**
I just needed to add documentation to the close and logout methods.

### CODE REVIEW

File: src/3box.js [hunk 1]
```diff
@@ -173,10 +173,18 @@ class ThreeBox {
     }
   }
 
+  /**
+   * Uninitializes the 3box. Should be called after you are done with
```
Comment (michaelsena) [inline src/3box.js]: Uninitializes the user's 3Box without clearing the local cache. Should be called after you are done using the 3Box instance, but without logging the user out.

File: src/3box.js [hunk 2]
```diff
@@ -173,10 +173,18 @@ class ThreeBox {
...
+   * the instance (unless logout has been called)
+   */
   async close () {
     await this.privateStore.close()
   }
 
+  /**
+   * Uninitializes the 3box and clears the local cache. If you call this
```
Comment (michaelsena) [inline src/3box.js]: Uninitializes the user's 3Box and clears local cache. If you call this, users will need to sign a consent message to log in the next time you call openBox.

File: src/3box.js (hunk 1 above)
Comment (c-castillo) [inline src/3box.js]: It can be unclear what `Uninitializes the 3box` actually means.

Comment (oed) [reply src/3box.js]: @c-castillo do you have a suggestion of what we could say instead? Basically we are just closing the orbitdb instance correctly.

Comment (c-castillo) [reply src/3box.js]: `Closes the 3box instance` or more technical `Closes the underlying storage instance of the 3box`
Maybe the first one is better

Comment (oed) [reply src/3box.js]: Ok, thanks!

**Comment by oed** [comment]
@michaelsena @c-castillo Pushed an update!

**Comment by c-castillo** [comment]
LGTM

**Pull request merged**